img/drawio/README.md

```diff
@@ -0,0 +1,6 @@
+In case the `*.png` files on the parent directory need to be adjusted, the `*.drawio.xml` files from this directory 
+should be imported into the `draw.io` online tool for editing.
+
+After editing:
+- export the `.png` file to replace the old one in the parent directory
+- export the `.drawio.xml` file to replace the old one on this directory
```

Comment:
@Sjors there you go

https://github.com/stratum-mining/sv2-spec/pull/98#issuecomment-2334123891